openlabcmd/openlabcmd/node.py

```diff
@@ -34,7 +31,6 @@ def to_zk_data(self):
         }
         return json.dumps(node_dict).encode('utf8')
 
-    @classmethod
-    def from_dict(cls, name, role, n_type, ip, heartbeat=None, alarmed=None,
-                  status=None):
-        return cls(name, role, n_type, ip, heartbeat, alarmed, status)
+    @ classmethod
+    def from_dict(cls, kwargs):
```

Comment:
not sure if `kwargs` is a good  input variable name. generally, it is used to be refer to `input keyword arguments`, but here you need to input a dict, right ? not similar to the above old function. I am also not very sure.

Reply:
Oh, `kwargs` is not a suitable name here. I'll change it.